src/tools/info/sys_info.c

```diff
@@ -176,7 +177,43 @@ static void print_sys_topo()
         print_row_separator(distance_width, name_width, num_devices, ' ', '|');
         print_row_separator(distance_width, name_width, num_devices, '-', '+');
     }
+
+    printf("#\n");
+    printf("# NUMA memory latency:\n");
+    printf("# To get optimal performance one should set the thread affinity based on system topology (NUMA nodes).\n"
+           "# Running ucx_info with the same affinity will give the memory latency estimation as it calculated by UCX.\n");
     printf("#\n");
+
+    /* Print table header */
```

Comment:
Code repetition (table above).
Can we feasibly put common code in a separate function?